aspnet/whitepapers/aspnet-and-iis6.md

```diff
@@ -177,25 +177,25 @@ To enable integrated authentication between SQL Server and ASP.NET, you will nee
 
 Open SQL Server Enterprise Manager (Start | Programs | Microsoft SQL Server | Enterprise Manager), select the appropriate server, and expand the Security folder:
 
-![](aspnet-and-iis6/_static/image22.jpg)
+![A screenshot of the Windows S Q L Enterprise Manager screen. The menu has Logins highlighted.](aspnet-and-iis6/_static/image22.jpg)
 
 If 'BUILTINT\IIS\_WPG' group is not listed, right-click on Logins and select 'New Login':
 
-![](aspnet-and-iis6/_static/image23.jpg)
+![A screenshot of the Windows S Q L Enterprise Manager SQL Server Login Properties screen with the General tab selected.](aspnet-and-iis6/_static/image23.jpg)
```

Comment:
Anything improves the living quality, and you have the budget, don’ wait and just go ahead.